packages/camera/camera_android/example/integration_test/camera_test.dart

```diff
@@ -205,6 +207,63 @@ void main() {
     expect(duration, lessThan(recordingTime - timePaused));
   });
 
+  testWidgets('Set description while recording', (WidgetTester tester) async {
+    final List<CameraDescription> cameras =
+        await CameraPlatform.instance.availableCameras();
+    if (cameras.length < 2) {
+      return;
+    }
+
+    final CameraController controller = CameraController(
+      cameras[0],
+      ResolutionPreset.low,
+      enableAudio: false,
+    );
+
+    await controller.initialize();
+    await controller.prepareForVideoRecording();
+
+    await controller.startVideoRecording();
+    sleep(const Duration(milliseconds: 500));
+
+    // set description while recording requires android >= 26
+    final DeviceInfoPlugin deviceInfo = DeviceInfoPlugin();
+    final AndroidDeviceInfo androidInfo = await deviceInfo.androidInfo;
+    final int sdk = androidInfo.version.sdkInt;
+    if (sdk < 26) {
```

Comment:
Instead of doing this, can you use `if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O)` for consistency across the plugin?

Reply:
This is in the Dart integration tests so I don' think I can do anything like that? 

What this line of code is doing is testing switching camera while recording, but expecting an error to be thrown if the sdk version is below 26, otherwise everything to work correctly.